README.md

```diff
@@ -334,6 +337,10 @@ You will see the following screen with the API endpoints:
 * You can also run the API to search the documents in the Solr server using the command below:
 ```docker compose exec full_text_searcher python main.py --env dev```
 
+According to the env use in the command line, the API will use the Solr URL in the configuration file `config_search.py`.
+If you use `--env htrc` the Solr URL will be `https://analytics.dev.htrc.indiana.edu/solr/core-1x/query`, however the
```

Comment:
How widely is HTRC exposing the interface to Solr? It _should_ be read-only but there is still the potential for some denial-of-service type things if the Solr API becomes directly exposed

Reply:
Regardless we might want to consider whether or not it's OK to include production hostnames in public documentation -- I'm not sure we ever really came to a clear consensus on that..